content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,217 @@
+===============
+Event templates
+===============
+
+The Odoo *Events* application provides users with the ability to customize and configure event
+templates, which can be used to expedite the event-creation process.
+
+These templates can be created and personalized in the application, and then selected from an event
+form, in order to quickly apply a series of settings and elements to the new event, all of which can
+be further modified, if needed.
+
+Event templates page
+====================
+
+In the Odoo *Events* application, event templates can quickly be created and modified.
+
+To begin, navigate to :menuselection:`Events app --> Configuration --> Event Templates`. Doing so
+reveals the :guilabel:`Event Templates` page. Here, users can find all the existing event templates
+in the database.
+
+.. image:: event_templates/event-templates-page.png
+   :align: center
+   :alt: The Event Templates page in the Odoo Events application.
+
+By default, Odoo provides users with three pre-configured event templates: :guilabel:`Exhibition`,
+:guilabel:`Training`, and :guilabel:`Sport`. All of these pre-configured event templates that Odoo
+provides by default have their own unique customizations applied to them.
+
+To change how these event templates appear on the *Template* drop-down field on an event form, users
+can drag-and-drop them into any desired order, using the :guilabel:`six block` icons, located to the
+left of each event template line on the :guilabel:`Event Templates` page.
+
+.. seealso::
+   To learn more about event forms, refer to the :doc:`create_events` documentation.
+
+Create event template
+=====================
+
+There are two ways users can create and configure an event template in Odoo *Events*.
+
+Either navigate to :menuselection:`Events app --> Configuration --> Event Templates`, and click the
+:guilabel:`New` button in the upper-left corner. Doing so reveals a blank event template form that
+can be customized in a number of different ways.
+
+Or, on an event form, start typing the name of a new event template in the *Template* field, and
```

Comment:
Suggestion around grammar and presentation:
I'm pretty sure if we begin a sentence with the conjunction "either," we need to pair it with another conjunction like "or" since the word _either_ is meant to present two (or more) options.


Alternatively, these options can be put in numbered list (without the either/or condition), with a quick **bold label** in the beginning, for a different type of presentation:
```suggestion
#. **On the dashboard**, by navigating to :menuselection:`Events app --> Configuration --> Event
   Templates` and clicking the :guilabel:`New` button in the upper-left corner. Doing so reveals a
   blank event template form that can be customized in a number of different ways.
#. **On an event form itself.** Start by typing the name of a new event template in the *Template*
   field, and
```